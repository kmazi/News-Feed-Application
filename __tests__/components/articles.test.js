import React from 'react';
import ReactDom from 'react-dom';
import Article from './../../app/components/articles.jsx';

describe('The headlines component', () => {
  it('must render without throwing error', () => {
    const div = document.createElement('div');
    ReactDom.render(<Article/>, div);
  });
});
