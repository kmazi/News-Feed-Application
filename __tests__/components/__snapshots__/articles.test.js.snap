// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`undefined 1`] = `
<div
  className="col-md-9"
  id="news-headline"
>
  <div
    className="row"
    data-content="news-header"
  >
    <h3
      className="pull-left"
    >
      <span
        className="badge"
      >
        
      </span>
      Headlines from 
      CNN
    </h3>
  </div>
  <div>
    <h4 />
  </div>
</div>
`;
